docs/pt_BR/README.md

```diff
@@ -0,0 +1,82 @@
+* Instalado [vue-devtools](https://github.com/vuejs/vue-devtools) e [devtron](https://github.com/electron/devtron) ferramentas para desenvolvimento
+* Habilidade para facilmente compilar e criar o arquivo final para instalar sua aplicação usando [electron-packager](https://github.com/electron-userland/electron-packager) ou [electron-builder](https://github.com/electron-userland/electron-builder)\*
+* `appveyor.yml` e `.travis.yml` configurados para automatizar o deploy com [electron-builder](https://github.com/electron-userland/electron-builder)\*
+* Habilidade para produzir saidas para browsers
+* Acessivel [NPM scripts](/npm_scripts.md)
+* Usa o [webpack](https://github.com/webpack/webpack) e [vue-loader](https://github.com/vuejs/vue-loader) com Hot Module Replacement
+* Automatização do reinicio dos processos `main` do electron
+* HTML/CSS/JS pre-processor suporte com [vue-loader](https://github.com/vuejs/vue-loader/)
+* ES6 com [`stage-0`](https://babeljs.io/docs/plugins/preset-stage-0/) by default
+* Uso de [`babili`](https://github.com/babel/babili) para remover a nescessidade de fazer transpiling completo para o ES5
+* ESLint \(com suporte ao [`standard`](https://github.com/feross/standard) e [`airbnb-base`](https://github.com/airbnb/javascript)\)\*
+* Teste unitario com \(with [`karma`](https://github.com/karma-runner/karma) + [`mocha`](https://github.com/mochajs/mocha)\)\*
+* End-to-end Testing \(with [`spectron`](https://github.com/electron/spectron) + [`mocha`](https://github.com/mochajs/mocha)\)\*
+
+\*Customizavel durante o uso da `vue-cli`
+
+### Introdução
+
+Esse template foi construído utilizando [`vue-cli`](https://github.com/vuejs/vue-cli) e inclui opções para customizar a estrutura final do seu app. O template utiliza `node@^7` ou superior como mínimo. electron-vue também recomenda utilizar o [`yarn`](https://yarnpkg.org), gerenciador de pacotes que manipula seus pacotes de uma forma muito melhor e vai ajudar a reduzir o peso final do projeto `yarn clean`.
+
+```bash
+# Instala o vue-cli e o template do projeto
+npm install -g vue-cli
+vue init simulatedgreg/electron-vue my-project
+
+# Instala as dependencias e executa seu app (your app)
+cd my-project
+yarn # ou npm install
+yarn run dev # ou npm run dev
+```
+
+##### Você e um usuario de Windows
+
+Por favor olhe a [**Uma Nota Para Usuarios de Windows**](https://simulatedgreg.gitbooks.io/electron-vue/content/en/getting_started.html#a-note-for-windows-users) para que você tenha certeza que esta com todas as dependencias instaladas para conseguir fazer deploy e construir a aplicação.
+
+##### Querendo usar Vue 1?
+
+So um ponto a ser dado, a suporte ao Vue 1 foi oficialmente descontinuado então a estrutura do projeto, features e documentação vão refletir essas mudanças \([**documentação legado**](https://github.com/SimulatedGREG/electron-vue/tree/1.0/docs)\).
```

Comment:
- Change `So um ponto a ser dado, a suporte ao Vue 1 foi oficialmente descontinuado então a estrutura do projeto` to `Aponte para o branch 1.0, o suporte ao Vue 1 foi oficialmente descontinuado, a estrutura do projeto`